sql/dql/basic-queries/where-3.md

```diff
@@ -32,36 +32,65 @@ aspects:
 
 ### `LIKE`, `IN`
```

Comment:
some learn more links with better explanations would be helpful here
e.g. there should be one about the in-depth LIKE operators and all its options